src/utils/is-zero-value-string.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Check if the value is a zero value string like "0px" or "0%"
+ */
+export const isZeroValueString = (v: string) => /^0[\S]+$/.test(v)
```

Comment:
What is the `\S` character in a regex?

Reply:
`\S` matches any non-whitespace characters. The regex here translates to "starts with 0 and doesn't contain any space until the end of the line".